lib/cn/spill.c

```diff
@@ -180,394 +180,439 @@ kv_spill_abort(struct cn_tstate_omf *omf, void *arg)
 {
 }
 
-/**
- * kv_spill() - merge key-value streams, then partition by child
- * Requirements:
- *   - Each input iterator must produce keys in sorted order.
- *   - Iterator iterv[i] must contain newer entries than iterv[i+1].
- */
-static merr_t
-kv_spill(struct cn_compaction_work *w)
+static inline bool
+is_spill_to_intnode(struct cn_tree_node *pnode, uint child)
```

Comment:
Should this function be renamed since internal nodes aren't a thing anymore?

Reply:
Oh I see this is a frankenstein sort of thing for now.

Reply:
Right, this will be gone.